client/src/components/TreeImageScrubber.js

```diff
@@ -142,24 +156,22 @@ function TreeImageScrubber({ classes, getScrollContainerRef, ...props }) {
     loadMoreTreeImages();
   };
 
+  scrollContainerRef = getScrollContainerRef() ? getScrollContainerRef() : null;
```

Comment:
This is a weird idiom (I don't think you wrote this). does `getScrollContainerRef()` not return null? Does it instead return undefined?